<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="6" hostname="Admins-Mac-mini.local" name="com.crm.qa.testcases.CreateDraftTest" tests="7" failures="0" timestamp="2023-08-29T10:40:04 IST" time="0.006" errors="1">
  <testcase name="CreateDraftTesting" time="0.006" classname="com.crm.qa.testcases.CreateDraftTest">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.crm.qa.testcases.CreateDraftTest.CreateDraftTesting(CreateDraftTest.java:43)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1259)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- CreateDraftTesting -->
  <system-out/>
  <testcase name="EmailDraftTest" classname="com.crm.qa.testcases.CreateDraftTest">
    <skipped/>
  </testcase> <!-- EmailDraftTest -->
  <system-out/>
  <testcase name="SubjectClickTest" classname="com.crm.qa.testcases.CreateDraftTest">
    <skipped/>
  </testcase> <!-- SubjectClickTest -->
  <system-out/>
  <testcase name="CloseComposeTest" classname="com.crm.qa.testcases.CreateDraftTest">
    <skipped/>
  </testcase> <!-- CloseComposeTest -->
  <system-out/>
  <testcase name="DraftBodyTest" classname="com.crm.qa.testcases.CreateDraftTest">
    <skipped/>
  </testcase> <!-- DraftBodyTest -->
  <system-out/>
  <testcase name="ToDraft" classname="com.crm.qa.testcases.CreateDraftTest">
    <skipped/>
  </testcase> <!-- ToDraft -->
  <system-out/>
  <testcase name="CCDraftTest" classname="com.crm.qa.testcases.CreateDraftTest">
    <skipped/>
  </testcase> <!-- CCDraftTest -->
  <system-out/>
</testsuite> <!-- com.crm.qa.testcases.CreateDraftTest -->
